packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -1067,6 +1086,100 @@ contract VotingV2 is
         return spamDeletionProposals[spamDeletionRequestId];
     }
 
+    /**
+     * @notice Enables voters to signal to pass a price request without using the standard commit/reveal schelling point
+     * mechanism. Intended to be used in the case that that the DVM irrecoverably broken (say the commit reveal
+     * mechanism is broken). By signaling on an emergency action, we can bypass this and still get a price set to allow
+     * the Governor to still execute governance actions to save the system.
+     * @dev by calling this, we modify the stakers pendingUnstake and unstakeRequestTime to prevent them from requesting
+     * or executing unstakes while singled on emergency actions.
+     * @param identifier the price identifier to signal on.
+     * @param actionTime the action time to signal on.
+     * @param ancillaryData the ancillary data to signal on.
+     */
+    function signalOnEmergencyAction(
+        bytes32 identifier,
+        uint256 actionTime,
+        bytes memory ancillaryData
+    ) public {
+        EmergencyAction storage action = emergencyActions[_encodePriceRequest(identifier, actionTime, ancillaryData)];
+        require(action.accountSignaled[msg.sender] == 0, "Already signaled");
```

Comment:
I don't think it's a problem, but it's important to know that if someone wants to stake more to increase his vote and save the contract, he shouldn't use the same wallet.

Reply:
or cancel and re-stake